python/mp/rosary.py

```diff
@@ -18,7 +19,7 @@
 class Bead:
     """Bead represents a single rosary bead."""
     def __init__(self, index=0):
-        self.index = index
+        self.index = int(index)
```

Comment:
this was an early experiment to speed things up, b/c I read that python3 iterators are slower if you don't force values to int().